File: 백준/Bronze/2869. 달팽이는 올라가고 싶다/달팽이는 올라가고 싶다.js
const [A,B,V] = require('fs').readFileSync('/dev/stdin').toString().trim().split(' ');
const day = A-B; // 올라가는 높이 - 내려가는 높이
let result = Math.ceil((V-A)/day) + 1; 

console.log(result);

File: 백준/Bronze/2869. 달팽이는 올라가고 싶다/달팽이는 올라가고 싶다.py
import sys
import math

input = sys.stdin.readline
A,B,V = list(map(int, input().strip().split()))

day = A - B # 올라가는 높이 - 내려가는 높이
result = math.ceil((V-A) / day) + 1
sys.stdout.write(f"{result}\n")